frontend/src/styles/material.sass

```diff
@@ -79,12 +80,12 @@ snack-bar-container
 .mat-form-field-wrapper
     padding-bottom: 5px
 
-.mat-radio-button.mat-accent.mat-radio-checked .mat-radio-outer-circle
-    border-color: $royal-blue
+::ng-deep .mat-radio-button.mat-accent.mat-radio-checked .mat-radio-outer-circle
+    border-color: $midnight
```

Comment:
I believe we don't need ng-deep when redefining styles in material.sass

Reply:
fix